test/compiler.test.js

```diff
@@ -2187,6 +2187,26 @@ describe('compiler', function() {
         'sourceFile', coffee);
     });
 
+    it('ignores sourcmap files when loading middleware',
+    function() {
+      var middleware = appdir.writeFileSync('my-middleware.js',
+        '// I am the middleware');
+      var sourcemap = appdir.writeFileSync('my-middleware.js.map',
+        '// I am a sourcemap');
+      appdir.writeConfigFileSync('middleware.json', {
+        'routes': {
+          './my-middleware': {},
+        },
+      });
+
+      var instructions = boot.compile(appdir.PATH);
+
+      expect(instructions.middleware.middleware[0]).have.property(
+        'sourceFile', middleware);
+      expect(instructions.middleware.middleware[0]).not.have.property(
```

Comment:
This second expectation is redundant - if the `sourceFile` property is set to `middleware`, then the second expectation will always pass. If `sourceFile` is not set to `middleware`, then the first `expect` will abort the test and this second expectation won't be run at all.

Please remove these two lines (2206-2207).

Reply:
Ah, yes, fair point. My goal was to ensure that the map file is definitely never loaded but I missed the index being used. Will remove.